tests/publiccloud/transfer_repos.pm

```diff
@@ -15,17 +15,16 @@ use testapi;
 use strict;
 use utils;
 use publiccloud::ssh_interactive "select_host_console";
-use publiccloud::utils qw(is_hardened);
+use version_utils qw(is_sle_micro);
 use maintenance_smelt qw(is_embargo_update);
 
 sub run {
     my ($self, $args) = @_;
     select_host_console();    # select console on the host, not the PC instance
-
     my $remote = $args->{my_instance}->username . '@' . $args->{my_instance}->public_ip;
     my @addons = split(/,/, get_var('SCC_ADDONS', ''));
     my $skip_mu = get_var('PUBLIC_CLOUD_SKIP_MU', 0);
-    my $repodir = "/opt/repos/";
+    my $repodir = (is_sle_micro) ? "/opt/repos/" : "/tmp/repos/";
```

Comment:
1. I thought we already proved that this switch to `/opt` is unrelated to failure which we have 
2. Why `is_sle_micro` and not `is_hardened` ? failures happens only with Hardened images anyway

Reply:
Considering https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18500#issuecomment-1905713972, here I let the previous existing code to run without refactoring, adding needed update for slem cases only